src/main/scala/no/ndla/learningpathapi/service/ConverterServiceComponent.scala

```diff
@@ -60,7 +60,7 @@ trait ConverterServiceComponent {
     }
 
     def asCoverPhoto(metaUrl: String, imageMeta: Option[ImageMetaInformation]): Option[api.CoverPhoto] = {
-      imageMeta.flatMap(_.images.full).map(full => api.CoverPhoto(full.url, metaUrl))
+      imageMeta.map(_.url).map(url => api.CoverPhoto(url, metaUrl))
```

Comment:
Denne kan vel forenkles til `imageMeta.map(img => api.CoverPhoto(img.url, metaUrl))`. Tror ikke det er nødvendig med to mappinger her.